src/schemas/RowSchema.tsx

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+import {CellType, CommentType, Review_Stages} from '../components/TimeCardPage/types'; 
+
+const optionalNumber = z.union([z.undefined(), z.number()]) 
+
+const optionalMember = z.union([z.undefined(), z.object({
+    Start: optionalNumber, End: optionalNumber, Author: optionalNumber
+})]); 
+
+export const CommentSchema = z.object({
+    Author:z.string(), 
```

Comment:
I think it might be best to pass in a User object for the author of comments. The backend can handle grabbing that data, but ultimately on the front end, you'll likely want a name, an email, and a profile pic (which are all part of the User schema as we've defined it rn). Or, just have this be a uuid and then make a separate getUserProfile() call for that, but that could get arduous.

It also makes it reeeeeally difficult unless we have a specific 'add a comment' endpoint to just pass a name to the backend and then turn it into a uuid. From my understanding, right now, the `Author` attribute is the name here. Obviously the reverse is easy (i.e. the uuid is stored in Dynamo for the author, and the backend gets the full user data from that and passes it to the frontend when sending over timesheet data).

Reply:
I think David's PR also has UserSchema defined for frontend if you want to use that

Reply:
Okay, one last thought on this:

In the getTimesheet endpoint, we could pass back a map of `uuid` to `User` objects that are needed in the body of the response (in addition to the actual list of entries). Everything in the timesheet entries will refer to a `uuid`, and the frontend can essentially just grab the corresponding `User` object anywhere they need it. 

Essentially:

`Response Body:
{
Entries: [],
UserMap: {
'abc': User1,
'xyz': User2,
etc. }
}
`

I think that's probably the best pattern I can think of to avoid data duplication - but also, this stuff is so tiny and we are dealing with such small amounts of user data that it probably doesn't really matter how we are passing it back and forth.